aibolit/patterns/classic_getter/classic_getter.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+from aibolit.ast_framework import ASTNodeType, AST
+from aibolit.ast_framework.ast_node import ASTNode
+
+
+class ClassicGetter:
+    '''
+    The method's name starts with get. Return statement is the only statement,
+    excepting asserts.
+    '''
+
+    def _check_body_nodes(self, check_setter_body: List[ASTNode]) -> bool:
+        '''
+        Check whether nodes are agree with the following types
+        (in self.suitable_nodes) or not.
+        '''
+        for node in check_setter_body:
+            has_expression_value = hasattr(node, "expression")
+            if not has_expression_value:
+                return False
+
+            is_return = node.node_type == ASTNodeType.RETURN_STATEMENT
+            is_expression_memeber_ref = node.expression.node_type == ASTNodeType.MEMBER_REFERENCE
+            if is_return and is_expression_memeber_ref:
+                return True
+
+        return False
```

Comment:
Why don't you directly check if body consist of one node, it is `RETURN_STATEMENT` and its expression type is `MEMBER_REFERENCE`? Looks much simpler and no needs for checking attributes.

Reply:
because it can contain assert, not only 1 statement